Makefile

```diff
@@ -1,11 +1,17 @@
 SHELL := /bin/bash
 
+
 BUNDLE_VERSION = 4.6.3
 BUNDLE_EXTENSION = crcbundle
 CRC_VERSION = 1.19.0
 COMMIT_SHA=$(shell git rev-parse --short HEAD)
 CONTAINER_RUNTIME ?= podman
 
+ifdef OKD_VERSION
```

Comment:
what if we add a ?:

```
BUNDLE_VERSION ?= 4.6.3
CRC_VERSION ?= 1.19.0
OKD ?= false
``` 

So when you build you do:
``` 
BUNDLE_VERSION=foo CRC_VERSION=bar OKD=true
```

Reply:
I like that idea.  I'll do that in a subsequent PR.